codex-rs/core/src/command_safety/is_safe_command.rs

```diff
@@ -49,19 +49,34 @@ fn is_safe_to_call_with_exec(command: &[String]) -> bool {
     {
         #[rustfmt::skip]
         Some(
+            "base64" |
```

Comment:
It's possible to specify an output file for `base64` with the `-o` flag. Would lead to an unintended write, assuming safe commands are supposed to be read-only

e.g. `base64 -d -o /tmp/x payload` bypasses approval even though it mutates the filesystem

BSD docs: https://ss64.com/mac/base64.html